pkgs/servers/mail/mailman/web.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, serverEMail ? "postmaster@example.org"
+, archiverKey ? "SecretArchiverAPIKey"
+, allowedHosts ? []
+}:
+
+let
+
+  allowedHostsString = stdenv.lib.concatMapStringsSep ", " (x: "\""+x+"\"") allowedHosts;
+
+in
+
+stdenv.mkDerivation {
+  name = "mailman-web-0";
+
+  unpackPhase = ":";
+
+  installPhase = ''
```

Comment:
these could be moved to `$out/${python.sitePackages}`

Reply:
86f8895abbfdb3ce118e0ca971f56c7f72871e29